readme.txt

```diff
@@ -225,19 +225,22 @@ Remember to always make a backup of your database and files before updating!
 * Tweak - Add back missing JSON-LD to Single Events. [TEC-3262]
 * Tweak - Minify the Freemius svg assets. [TEC-3215]
 * Tweak - Remove "(beta)" label from URL source type of import. [TEC-3289]
+* Tweak - Remove Eventbrite dependency on Aggregator Admin view. [EA-133]
 * Tweak - Reword the Filter Activation setting's labels for better clarity.  [FBAR-79]
 * Fix - Ensure that any values for `liveFilterUpdate` that are not `automatic` or `manual` get converted. [FBAR-83]
 * Fix - Breakpoint code compatibility with the latest version of jQuery 3.4.1 [BTRIA-154]
 * Fix - Remove numbers from listed events in event list widget for Genesis theme. [TEC-2957]
 * Fix - Customizer accent color affects featured line and text. [TEC-3277]
 * Fix - Resolve conflict between datepicker holding consistent size when navigating and flexibility on small screen sizes. [TEC-3322]
 * Fix - Respect EA `post_status` from settings when an event does not have a defined value. [TEC-3255]
+* Fix - Save the `post_status` for a scheduled import when using a different value other than `publish` [TEC:3296]
```

Comment:
We use the `-` to stick w/ the same way the tickets are used in the URL; it makes easier to review changelogs and copy-paste the ticket numbers in the URL.
```suggestion
* Fix - Save the `post_status` for a scheduled import when using a different value other than `publish` [TEC-3296]
```